examples/ExportSystemsToCSV/exportSystemsToCSV.go

```diff
@@ -2,36 +2,78 @@ package main
 
 import (
 	"encoding/csv"
+	"flag"
 	"fmt"
-	"github.com/TheJumpCloud/jcapi"
+	"log"
 	"os"
+
+	"github.com/TheJumpCloud/jcapi"
+	jcapiv2 "github.com/TheJumpCloud/jcapi-go/v2"
 )
 
 const (
-	apiUrl string = "https://console.jumpcloud.com/api"
+	apiUrlDefault string = "https://console.jumpcloud.com/api"
 )
 
 func main() {
-	apiKey := os.Getenv("JUMPCLOUD_APIKEY")
+	var apiKey string
+	var apiUrl string
+
+	// Obtain the input parameters: api key and url (if we want to override the default url)
+	flag.StringVar(&apiKey, "key", "", "-key=<API-key-value>")
+	flag.StringVar(&apiUrl, "url", apiUrlDefault, "-url=<jumpcloud-api-url>")
+	flag.Parse()
+
+	// if the api key isn't specified, try to obtain it through environment variable:
+	if apiKey == "" {
+		apiKey = os.Getenv("JUMPCLOUD_APIKEY")
```

Comment:
Might want to make `JUMPCLOUD_APIKEY` a constant so we don't have any typos anywhere in help text, etc.